src/main/kotlin/in/rcard/assertj/arrowcore/AbstractEitherAssert.kt

```diff
@@ -81,6 +82,12 @@ abstract class AbstractEitherAssert<
         return myself
     }
 
+    fun hasRightValueSatisfying(requirement: Consumer<RIGHT>): SELF {
```

Comment:
You can change the signature with the following:
```kotlin
fun hasRightValueSatisfying(requirement: (RIGHT) -> Unit): SELF
```
It's more Kotlinish.
Apply the same for the method `hasLeftValueSatisfying`.